modules/compute/bootstrap.sh.tpl

```diff
@@ -47,7 +47,7 @@ echo "Consul installation complete."
 
 # Get VM private ip address
 INSTANCE_PRIVATE_IP=$(ifconfig eth0 | grep "inet addr" | awk '{ print substr($2,6) }')
-
+INSTANCE_PUBLIC_IP=$(dig +short myip.opendns.com @resolver1.opendns.com)
```

Comment:
I'm already passing in `var.public_ip` to the `template_file` resource and passing that to the template as `join_ip = "${var.public_ip[0]}"`. Do you prefer for me to use your method or my method?